lib/views/wallet/wallet_page/common/coins_list_header.dart

```diff
@@ -73,29 +73,56 @@ class _CoinsListHeaderDesktop extends StatelessWidget {
       );
     }
 
+    // ...existing code...
+
     return Container(
       padding: const EdgeInsets.fromLTRB(20, 8, 20, 8),
       child: Row(
         children: [
           // Asset header
-          Text(LocaleKeys.asset.tr(), style: style),
-
-          const Spacer(flex: 4),
-
-          // Balance header
-          Text(LocaleKeys.balance.tr(), style: style),
+          Expanded(
+            flex: 3,
+            child: Text(
+              LocaleKeys.asset.tr(),
+              style: style,
+            ),
+          ),
 
-          const Spacer(flex: 2),
+          // Price header
+          Expanded(
+            flex: 2,
+            child: Text(
+              LocaleKeys.price.tr(),
+              style: style,
+            ),
+          ),
 
           // 24h change header
-          Padding(
-            padding: const EdgeInsets.only(right: 48),
-            child: Text(LocaleKeys.change24hRevert.tr(), style: style),
+          Expanded(
+            flex: 2,
+            child: Text(
+              LocaleKeys.change24hRevert.tr(),
+              style: style,
+            ),
+          ),
+
+          // Chart header
+          Expanded(
+            flex: 2,
+            child: Text(
+              LocaleKeys.chart.tr(),
+              style: style,
+            ),
           ),
 
-          const Spacer(flex: 2),
+          // Space for expand button
+          ConstrainedBox(
+            constraints: const BoxConstraints(minWidth: 48),
+            child: const SizedBox(),
+          ),
         ],
       ),
     );
+// ...existing code...
```

Comment:
I'm sorry Dave, I can't do that.
![image](https://github.com/user-attachments/assets/2930dd36-b807-43e8-b6f6-7ae86f2c3ea6)